### PR TITLE
Replacing cbtext with countryball.description

### PR DESCRIPTION
**Pull request by MaxxMXSZ**
Just to balance everything else so it looks the same in anyway

### CODE REVIEW

File: ballsdex/packages/balls/cog.py [hunk 1]
```diff
@@ -439,19 +439,17 @@ async def favorite(self, interaction: discord.Interaction, countryball: BallInst
 
             countryball.favorite = True  # type: ignore
             await countryball.save()
-            emoji = self.bot.get_emoji(countryball.countryball.emoji_id) or ""
             await interaction.response.send_message(
-                f"{emoji} `#{countryball.pk:0X}` {countryball.countryball.country} "
+                f"{countryball.description(include_emoji=True, bot=self.bot, is_trade=True)} "
```
Comment (flaree) [inline ballsdex/packages/balls/cog.py:443]: any reason why is_trade kwarg is set to true?

File: ballsdex/packages/balls/cog.py [hunk 2]
```diff
@@ -439,19 +439,17 @@ async def favorite(self, interaction: discord.Interaction, countryball: BallInst
...
                 ephemeral=True,
             )
 
         else:
             countryball.favorite = False  # type: ignore
             await countryball.save()
-            emoji = self.bot.get_emoji(countryball.countryball.emoji_id) or ""
             await interaction.response.send_message(
-                f"{emoji} `#{countryball.pk:0X}` {countryball.countryball.country} "
+                f"{countryball.description(include_emoji=True, bot=self.bot, is_trade=True)} "
```
Comment (flaree) [inline ballsdex/packages/balls/cog.py:452]: same here

File: ballsdex/packages/balls/cog.py [hunk 3]
```diff
@@ -527,11 +525,10 @@ async def give(
         trade = await Trade.create(player1=old_player, player2=new_player)
         await TradeObject.create(trade=trade, ballinstance=countryball, player=old_player)
 
-        cb_txt = countryball.description(
-            short=True, include_emoji=True, bot=self.bot, is_trade=True
-        )
         await interaction.response.send_message(
-            f"You just gave the {settings.collectible_name} {cb_txt} to {user.mention}!"
+            "You just gave "
+            f"{countryball.description(include_emoji=True, bot=self.bot, is_trade=True)} "
```
Comment (flaree) [inline ballsdex/packages/balls/cog.py:530]: and here

**Review by flaree — CHANGES_REQUESTED** [review]
small changes to be addressed

File: ballsdex/packages/balls/cog.py [hunk 4]
```diff
@@ -527,11 +525,10 @@ async def give(
...
         await TradeObject.create(trade=trade, ballinstance=countryball, player=old_player)
 
-        cb_txt = countryball.description(
-            short=True, include_emoji=True, bot=self.bot, is_trade=True
-        )
         await interaction.response.send_message(
-            f"You just gave the {settings.collectible_name} {cb_txt} to {user.mention}!"
+            "You just gave "
+            f"{countryball.description(include_emoji=True, bot=self.bot)} "
+            "to {user.mention}!"
```
Comment (laggron42) [inline ballsdex/packages/balls/cog.py:539]: I'm not sure what you did here? It looks the same as before, except you removed the `short` and `is_trade` arguments which make sense here.

You also forgot an `f` for your string, `{user.mention}` isn't processed at the last line

**Review by laggron42 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**